<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFeatureDialog</class>
 <widget class="QDialog" name="CreateFeatureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_entityType">
         <property name="text">
          <string>type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_entityType"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
         <item>
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,3">
         <item>
          <widget class="QLabel" name="label_group">
           <property name="text">
            <string>group: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_group">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,3">
         <item>
          <widget class="QLabel" name="label_count">
           <property name="text">
            <string>count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_count">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,3">
         <item>
          <widget class="QLabel" name="label_function">
           <property name="text">
            <string>function:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_function">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_common">
       <property name="text">
        <string>common</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_actual">
       <property name="text">
        <string>actual</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_nominal">
       <property name="text">
        <string>nominal</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_mConfig">
         <property name="text">
          <string>measurement config:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_mConfig"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_nominalSystem">
         <property name="text">
          <string>nominal system</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_nominalSystem"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3">
         <item>
          <widget class="QLabel" name="label_startSystem">
           <property name="text">
            <string>start system</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_startSystem"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,3">
         <item>
          <widget class="QLabel" name="label_destinationSystem">
           <property name="text">
            <string>destination system</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_destinationSystem"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_cancel">
         <property name="text">
          <string>cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_ok">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
